MonstieWash/Assets/Scripts/Consumables/ConsumablesManager.cs

```diff
@@ -36,47 +36,34 @@ public enum UiState
         Closed
     }
 
-    [Serializable]
-    private class consumableData
-    {
-        [SerializeField] private Consumable consumable;
-        [SerializeField] private int quantity;
-
-        public Consumable Consumable { get { return consumable; } }
-        public int Quantity 
-        { 
-            get { return quantity;  }
-            set { quantity = value; }        
-        }  
 
-        public consumableData(Consumable type, int quant)
-        {
-            consumable = type;
-            quantity = quant;
-        }
-    }
 
     public void OnEnable()
     {
-        InputManager.Instance.OnActivate += CheckClickedOn;
+        //InputManager.Instance.OnActivate += CheckClickedOn;
     }
 
     public void OnDisable()
     {
-        InputManager.Instance.OnActivate -= CheckClickedOn;
+        //InputManager.Instance.OnActivate -= CheckClickedOn;
     }
```

Comment:
Just to keep things clean I'd prefer if these methods were removed altogether since they are no longer being used. (unless you plan to re-add stuff in here later on)